core/docs/API-Changelog-0.1.0.txt

```diff
@@ -0,0 +1,424 @@
+---------------------------------
+Terminology
+---------------------------------
+
+[A]: Added
+[R]: Removed
+[C]: Changed
+[D]: Deprecated
+[O]: Old
+[N]: New
+
+Also note that the indentation forms a naturual grouping.
+
+---------------------------------
+API
+---------------------------------
+
+[A] Streamly.Unicode.String
+    [A] str :: QuasiQuoter
+[A] Streamly.Unicode.Stream
```

Comment:
In the changelog of the modules that were moved from streamly to streamly-core we should note only new additions instead of all functions.